snippets/fsh-snippets.json

```diff
@@ -16,21 +16,47 @@
 		"prefix": ["ext", "Ext"],
 		"body": [
 			"Extension: $1",
-			"Id: ${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}",
-			"Title: \"${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}\"",
-			"Description: \"$2\"",
+			"Id: ${2:${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}}",
+			"Title: \"${3:${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}}\"",
+			"Description: \"$4\"",
 			"* $0"
 		],
 		"description": "Create a FSH Extension"
 	},
 
+	"FSH Logical": {
+		"prefix": ["log", "Log"],
+		"body": [
+			"Logical: $1",
+			"Parent: ${2:Base}",
+			"Id: ${3:$1}",
+			"Title: \"${4:${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}}\"",
+			"Description: \"$5\"",
+			"* $0"
+		],
+		"description": "Create a FSH Resource"
+	},
+
+	"FSH Resource": {
+		"prefix": ["res", "Res"],
+		"body": [
+			"Resource: $1",
+			"Parent: ${2|DomainResource,Resource|}",
+			"Id: ${3:$1}",
```

Comment:
Similar to the question above, is this usually the same as the Resource name or does it also get kebab cased like the Profile and Extension ids?

Reply:
As noted in my other comment, I think it's usually the same.